pallets/subtensor/src/math.rs

```diff
@@ -2254,4 +2254,141 @@ mod tests {
         assert_vec_compare( &matmul( &w, &vec![ I32F32::from_num(2.0); 3] ), &vec![ I32F32::from_num(6),  I32F32::from_num(12),  I32F32::from_num(18)], epsilon );
     }
 
-}
\ No newline at end of file
+    #[test]
+    fn test_math_fixed_to_u16() {
+        let expected = u16::MIN;
+        assert_eq!(fixed_to_u16(I32F32::from_num(expected)), expected);
+
+        let expected = u16::MAX / 2;
+        assert_eq!(fixed_to_u16(I32F32::from_num(expected)), expected);
+
+        let expected = u16::MAX;
+        assert_eq!(fixed_to_u16(I32F32::from_num(expected)), expected);
+    }
+
+    #[test]
+    #[should_panic(expected = "overflow")]
+    fn test_math_fixed_to_u16_panics() {
+        let bad_input = I32F32::from_num(u32::MAX);
+        fixed_to_u16(bad_input);
+
+        let bad_input = I32F32::from_num(-1);
+        fixed_to_u16(bad_input);
+    }
+
+    // TODO: Investigate why `I32F32` and not `I64F64`
```

Comment:
`fixed_to_u64` doesn't really appear to be used anywhere in the codebase, and the `I32F32` choice seems to have followed the previous fn templates, but should ideally be `I64F64`.

Reply:
Thanks for confirming my suspicions!... I'll add fixing this to my future self's to-dos